pkg/gatherers/conditional/conditional_gatherer.go

```diff
@@ -86,8 +102,9 @@ type Gatherer struct {
 	gatherProtoKubeConfig   *rest.Config
 	metricsGatherKubeConfig *rest.Config
 	imageKubeConfig         *rest.Config
-	firingAlerts            map[string]bool // golang doesn't have sets :(
-	gatheringRules          []GatheringRule
+	// there can be multiple instances of the same alert
+	firingAlerts   map[string][]Alert // golang doesn't have sets :(
```

Comment:
Just a small nitpick: we can already delete "golang doesn't have sets :(" comment, it's already a proper map :)